src/pages/home/sidebar/ChatSwitcherView.js

```diff
@@ -428,7 +428,7 @@ class ChatSwitcherView extends React.Component {
                 alternateText: personalDetail.login,
                 searchText: personalDetail.displayName === personalDetail.login ? personalDetail.login
                     : `${personalDetail.displayName} ${personalDetail.login}`,
-                icon: Array.isArray(personalDetail.avatarURL) ? personalDetail.avatarURL : [personalDetail.avatarURL],
+                icons: _.isArray(personalDetail.avatarURL) ? personalDetail.avatarURL : [personalDetail.avatarURL],
```

Comment:
Still unclear why we need the check? When is this going to be an array?

Reply:
Yes. avatarURL is a string, not array.
I don't need to check it.